source/auth/tests/mongodb-aws.md

```diff
@@ -95,11 +95,11 @@ mongodb://localhost/?authMechanism=MONGODB-AWS
 ## AssumeRole
 
 Drivers MUST be able to authenticate using temporary credentials returned from an assume role request. These temporary
-credentials consist of an access key ID, a secret access key, and a security token passed into the URI. A sample URI
-would be:
+credentials consist of an access key ID, a secret access key, and a security token passed into the URI. Drivers MUST
```

Comment:
I think we should update the language here since it still says that these are "passed into the URI"

Reply:
Updated to note the values are in the environment